ontopy/excelparser.py

```diff
@@ -304,8 +307,12 @@ def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-bran
                     if force is True:
                         pass
```

Comment:
We want to issue a warning when `force` is true.
```suggestion
                        warnings.warn(
                            f"Error in Property assignment for: {concept}. "
                            f"Property to be Evaluated: {prop}. "
                            f"Error is {exc}."
                        )
```